partcad/src/partcad/adhoc/convert.py

```
@@ -27,12 +36,21 @@
+        pc_logging.info(f"Loaded input part: {input_path}")
+        pc_logging.info(f"Shape: {type(shape)}")
+
+        if part.errors:
+            raise RuntimeError(f"Failed to load the input part: {part.errors}")
 
         # Export the part to the desired output format
         export_part(part, output_filename, output_type, ctx)
-    except Exception:
-        raise RuntimeError("Failed to load the input part.")
+    except Exception as e:
+        raise RuntimeError(f"Failed to convert: {e.with_traceback(None)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)